contrib/scalapblib/src/mill/contrib/scalapblib/ScalaPBModule.scala

```diff
@@ -113,7 +113,7 @@ trait ScalaPBModule extends ScalaModule {
                 if (os.exists(protoDest))
                   Task.log.error(s"Warning: Overwriting ${dest} / ${os.SubPath(entry.getName)} ...")
                 Using.resource(os.write.over.outputStream(protoDest, createFolders = true)) { os =>
-                  IO.stream(zip, os)
+                  _root_.os.Internals.transfer(zip, os)
```

Comment:
Wondering, why this API is called internals. Sound like it should not be used outside. OTOS, we have a close channel to os-lib. :-D

Reply:
yeah, we're already using all sorts of experimental stuff from that library, so I guess it's fine haha